### PR TITLE
remove rowKeyElement dependencies test for code reivew

### PR DESCRIPTION
**Pull request by JackShi148**

### CODE REVIEW

File: src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java [hunk 1]
```diff
@@ -451,8 +451,18 @@ public ObTable getOdpTable() {
     private abstract class TableExecuteCallback<T> {
         private final Object[] rowKey;
 
+        //        private final Row row = new Row();
+
         TableExecuteCallback(Object[] rowKey) {
             this.rowKey = rowKey;
+            //            String[] columnNames = getRowKeyElement(tableName).keySet().toArray(new String[0]);
```
Comment (GroundWu) [inline src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java:458]: plz remove useless comments

File: src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java [hunk 2]
```diff
@@ -684,9 +694,9 @@ private <T> T executeMutation(String tableName, MutationExecuteCallback<T> callb
             long costMillis = currentExecute - startExecute;
             if (costMillis > runtimeMaxWait) {
                 throw new ObTableTimeoutExcetion("it has tried " + tryTimes
-                                                 + " times and it has waited " + costMillis
-                                                 + "/ms which exceeds response timeout "
-                                                 + runtimeMaxWait + "/ms");
+                        + " times and it has waited " + costMillis
```
Comment (GroundWu) [inline src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java:697]: it seems only change the formats ?

File: src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java (hunk 1 above)
Comment (JackShi148) [reply src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java:458]: fixed

File: src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java [hunk 3]
```diff
@@ -1337,40 +1347,40 @@ private String refreshTableNameByTableGroup(String physicalTableName, String tab
...
      * @return
      */
-    private long getPartition(TableEntry tableEntry, Object[] rowKey) {
-        // non partition
-        if (!tableEntry.isPartitionTable()
-            || tableEntry.getPartitionInfo().getLevel() == ObPartitionLevel.LEVEL_ZERO) {
-            return 0L;
-        }
+    private long getPartition(TableEntry tableEntry, Row row) {
+        // change up to getTable to judge
```
Comment (GroundWu) [inline src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java]: useless code?

Comment (JackShi148) [reply src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java]: fixed

File: src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java [hunk 4]
```diff
@@ -1462,8 +1456,33 @@ public ObPair<Long, ObTableParam> getTable(String tableName, Object[] rowKey, bo
...
                                                                                            throws Exception {
         TableEntry tableEntry = getOrRefreshTableEntry(tableName, refresh, waitForRefresh);
+        long partId;
+        if (!tableEntry.isPartitionTable()
+                || tableEntry.getPartitionInfo().getLevel() == ObPartitionLevel.LEVEL_ZERO) {
+            partId = 0L;
+        }
+        else {
+            Row row = new Row();
+            List<String> curTableRowKeyNames = new ArrayList<String>(tableEntry.getRowKeyElement().keySet());
```
Comment (GroundWu) [inline src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java:1466]: what are the differences between the curTableRowKeyNames and inputRowKeyNames ?

**Pull request closed without merging**